flowey/flowey_lib_hvlite/src/_jobs/build_and_publish_openvmm_hcl_baseline.rs

```diff
@@ -31,23 +31,25 @@ impl SimpleFlowNode for Node {
     fn process_request(request: Self::Request, ctx: &mut NodeCtx<'_>) -> anyhow::Result<()> {
         let Request { done, artifact_dir } = request;
 
-        let baseline_hcl_build = ctx.reqv(|v| build_openvmm_hcl::Request {
-            build_params: OpenvmmHclBuildParams {
-                target: CommonTriple::X86_64_LINUX_MUSL,
-                profile: OpenvmmHclBuildProfile::OpenvmmHclShip,
-                features: (OpenhclIgvmRecipe::X64)
-                    .recipe_details(OpenvmmHclBuildProfile::OpenvmmHclShip)
-                    .openvmm_hcl_features,
-                no_split_dbg_info: false,
-            },
-            openvmm_hcl_output: v,
-        });
-
-        ctx.req(artifact_openvmm_hcl_sizecheck::publish::Request {
-            openvmm_openhcl_x86: baseline_hcl_build,
-            artifact_dir,
-            done,
-        });
+        if let Some(artifact_dir) = artifact_dir {
```

Comment:
once you implement https://github.com/microsoft/openvmm/pull/909/files#r1970741447, you won't need this funky "_job is a noop in certain situations" thing anymore